js/plotsvy.js

```diff
@@ -40,6 +40,7 @@ function Plotsvy(targetDiv, inputURL) {
     parentDiv.append(htmlToElement(plotsvy_html))
     var chartDiv = parentDiv.querySelector('#chart_div')
     ui = new UI(parentDiv)
+    _this.ui = ui
```

Comment:
Do we need both local and global scope of the same object here?

Reply:
Good question, the short answer is no, the public (_this) scope is not needed for any functionality. 

I was planning on using it to create tests through `plotsvy.ui.<method-to-test>`, but on closer consideration it's likely better to test the modules on their own.

I'll add an update to remove it.